darwin/dataset/download_manager.py

```diff
@@ -59,7 +59,9 @@ def download_all_images_from_annotations(
 
     annotations_to_download_path = []
     for annotation_path in annotations_path.glob(f"*.{annotation_format}"):
-        annotation = json.load(annotation_path.open())
+        with annotation_path.open() as file:
+            annotation = json.load(file)
+            file.close()
```

Comment:
As far as I'm aware, files are automatically closed when control leaves the `with` block, so these `file.close()` calls can be avoided

Reply:
Done.